frontend/src/e2e-playwright/pages/employee/messages/messages-page.ts

```diff
@@ -53,4 +75,52 @@ export default class MessagesPage {
     await this.#sentMessagesBoxRow.click()
     await waitUntilEqual(() => this.existsSentMessage(), true)
   }
+
+  async draftNewMessage(title: string, content: string) {
+    await this.#newMessageButton.click()
+    await waitUntilEqual(() => this.isEditorVisible(), true)
+    await this.#inputTitle.fill(title)
+    await this.#inputContent.fill(content)
+    await this.#receiverSelection.click()
+    await this.page.keyboard.press('Enter')
+    // Draft is not saved immediately so we wait for a bit.
+    await this.page.waitForTimeout(500)
```

Comment:
Is there any indicator on the page that we could use? Or maybe something disappears that existed before pressing Enter; the editor itself? Random sleeps are always a bit iffy.

Reply:
There isn't, but it seems like it wouldn't be a huge task to add some kind of indicator. I'll do that.